### PR TITLE
Make loading screen into reusable component

### PR DESCRIPTION
**Pull request by amprokop**
connects https://github.com/department-of-veterans-affairs/caseflow/issues/2667

## Testing

Reader still displays spinner with message on pageload

### CODE REVIEW

File: client/app/components/LoadingScreen.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { loadingSymbolHtml } from './RenderFunctions';
+
+const LoadingScreen = ({ spinnerColor, message }) => {
```
Comment (NickHeiner) [inline client/app/components/LoadingScreen.jsx]: Can you make this a `PureComponent` so as to avoid superfluous renders?

**Comment by NickHeiner** [comment]
LGTM based on a quick scan of the code.

File: client/app/components/LoadingScreen.jsx (hunk 1 above)
Comment (amprokop) [inline client/app/components/LoadingScreen.jsx]: Can you explain this? I'm not sure I understand how that would avoid superfluous renders— perhaps our react practices have gotten away from me

Comment (NickHeiner) [inline client/app/components/LoadingScreen.jsx]: (Talked in Slack. Anyone else who wants to know the answer can ping me.)

**Comment by kierachell** [comment]
loadmebabyalltheway
:(

**Comment by amprokop** [comment]
@kierachell — hmm.. no CSS classes have changed in this PR. Have you 100% pinned this regression on this commit?

**Comment by kierachell** [comment]
Nothing's ever 100% for me cause I have no trust.
Found the fix, so we good.

**Pull request merged**